apps/common/src/python/mediawords/tag_from_annotation/fetch_and_tag.py

```diff
@@ -0,0 +1,403 @@
+import abc
```

Comment:
Heh, it turns out that there are API endpoints which return those raw CLIFF / NYTLabels annotations. So that was the reason why we were storing raw responses!

Good news is that I'm 99,998% positive that no one uses those raw responses for anything, and even if they were using those, it would be easier just to ask CLIFF / NYTLabels annotators directly for them to generate a new raw response.

So could you do the following:

1. Confirm with Dennis that those API calls (https://github.com/mediacloud/backend/blob/master/doc/api_2_0_spec/admin_api_2_0_spec.md#apiv2storiescliff and https://github.com/mediacloud/backend/blob/master/doc/api_2_0_spec/admin_api_2_0_spec.md#apiv2storiesnytlabels) are not used anywhere;
    * If they are, then `break`, ping me and I'll rewrite Perl API calls for them to get those raw annotations from the services which generate them.
    * If they are not, then `continue`
2. Delete everything that references `mediawords.annotator` Python module, including the module itself;
3. Delete everything that references `MediaWords::Annotator` Perl package, including the package itself. I realize Perl is not your speciality, but deleting a Perl package should be rather straightforward - get rid of https://github.com/mediacloud/backend/tree/master/apps/common/src/perl/MediaWords/Annotator and remove everything in code that references `MediaWords::Annotator` (including whole `sub cliff` and `sub nytlabels` subroutines that "serve" the API call). Let me know if you need my help with that.
4. Update the API spec (https://github.com/mediacloud/backend/blob/master/doc/api_2_0_spec/admin_api_2_0_spec.md) for it to no longer include the references to those two removed API calls.
    * You can use `./doc/update_toc.sh` to update Table of Contents of the API spec after adding / removing some sections.

Reply:
Regarding 1:

```
Dennis  < 1 minute ago
I can confirm that both are indeed used
```

😞 

No rush on the Perling, I can still get this branch ready in the meantime

Reply:
Darn it :( Thanks for asking Dennis. Okay, then I'll replace those two API endpoints with something that calls the `*-annotator` service and we could take it from there.